control/tests/bdalg_test.py

```diff
@@ -270,6 +272,46 @@ def test_feedback_args(self):
         sys = ctrl.feedback(1, frd)
         self.assertTrue(isinstance(sys, ctrl.FRD))
 
+    def testConnect(self):
```

Comment:
These tests only look for failures. What about tests that successfully connect systems?

Obviously I will pytest.parametrize this for #438

Reply:
> These tests only look for failures. What about tests that successfully connect systems?

good idea; added. 
 
> Obviously I will pytest.parametrize this for #438

thx!